packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -205,12 +205,24 @@ const DesignButton: React.FC< DesignButtonProps > = ( {
 		);
 	}
 
+	const previewDesign = () => {
+		if ( design.is_virtual && design.style_variation ) {
+			return onPreview( design, design.style_variation );
+		}
+		return onPreview( design );
+	};
+
+	const getTitle = () => {
+		if ( design.is_virtual && design.style_variation ) {
+			return `${ design.title } – ${ design.style_variation.title }`;
+		}
+
+		return design.title;
+	};
```

Comment:
As mentioned in D100722#2062749-code, we could do this in the endpoint.

Reply:
Already replied in the diff, but cross-posting here for visibility.

I actually did that first, but the `onPreview( design, styleVariation )` in the frontend opened the preview with a big "Theme - Title" title:

<img width="1275" alt="Screenshot 2023-02-09 at 11 28 45" src="https://user-images.githubusercontent.com/1233880/217789209-16901f9c-6578-4cf5-b56f-98361d1449aa.png">

That's because `design.title` comes with the title already built by the backend, so we'd need to add more logic in the frontend to "unbuild" the title. That seemed more complex / confusing, so I ultimately decided to build the title in the frontend.